packages/developer-portal/src/components/pages/login/login.tsx

```diff
@@ -30,18 +29,16 @@ export const handleShowNotificationAfterPasswordChanged = (
   }
 }
 
-export const onLoginButtonClick = (isFirtTimeLogin: boolean) => {
+export const onLoginButtonClick = () => {
   return () => {
-    const redirectRoute = getDefaultRoute(isFirtTimeLogin)
-    reapitConnectBrowserSession.connectLoginRedirect(redirectRoute)
+    reapitConnectBrowserSession.connectLoginRedirect(getDefaultRoute(true))
   }
 }
```

Comment:
I think you can remove closure here
```
export const onLoginButtonClick = () => {
  const redirectRoute = getDefaultRoute(isFirtTimeLogin)
  reapitConnectBrowserSession.connectLoginRedirect(redirectRoute)
}
```

Reply:
Same as Cuong comment

Reply:
That doesn't work because I always want to redirect to the apps page, not welcome. Welcome is handled in the private route now.